Extension/src/LanguageServer/client.ts

```diff
@@ -2239,7 +2239,7 @@ export class DefaultClient implements Client {
                 try {
                     promise.resolve(task ? await task() : undefined);
                 } catch (e) {
-                    console.log(e);
+                    log(e);
```

Comment:
The compiler wants this `e` to be a string -- maybe keep this as console.log?

Reply:
Kept this as-is.